docs/en/ingest-management/fleet/fleet-api-docs.asciidoc

```diff
@@ -0,0 +1,12 @@
+[[fleet-api-docs]]
+= {{fleet} API Docs}
```

Comment:
```suggestion
= {fleet} APIs
```

Reply:
: / lol, I wasn't sure how this was expecting to resolve.  i couldn't have made this pr any more complicated (and wrong) for how small it is.